### PR TITLE
[DOC] Troubleshooting Expensive Searches

### PR DESCRIPTION
**Pull request by stefnestor**
👋 re: https://github.com/elastic/elasticsearch/issues/73222 adds in content so we can link to users on how to find source of expensive searches.

### CODE REVIEW

**Review by abdonpijpelink — COMMENTED** [review]
Thanks @stefnestor . I've pushed a few edits and fixed the CI build error.

I don't understand the sentence _"The following is commonly used, only short-term `elasticsearch.yml` settings to trace queries"_. Are you saying these settings should only be enabled temporarily?

**Comment by stefnestor** [comment]
Ah, agreed. Thanks for reading through, @abdonpijpelink ! These settings will emit verbose logging, so users should disable it when not troubleshooting.

File: docs/reference/troubleshooting/troubleshooting-searches.asciidoc [hunk 1]
```diff
@@ -297,4 +297,24 @@ used by the data stream.
...
+for a data stream>>.
+
+[discrete]
+[[troubleshooting-slow-searches]]
+=== Find slow queries
+
+<<index-modules-slowlog,Slow logs>> can help pinpoint slow performing search 
+requests. Enabling <<auditing-settings,audit logging>> on top can help determine 
+query source. The following is commonly used, only short-term `elasticsearch.yml` 
+settings to trace queries:
```
Comment (abdonpijpelink) [inline docs/reference/troubleshooting/troubleshooting-searches.asciidoc:309]:
```suggestion
requests. Enabling <<auditing-settings,audit logging>> on top can help determine 
query source. Add the following settings to the `elasticsearch.yml` configuration file
to trace queries. The resulting logging is verbose, so disable these settings when not 
troubleshooting.
```

Comment (abdonpijpelink) [reply docs/reference/troubleshooting/troubleshooting-searches.asciidoc:309]: I've added a proposal to clarify the settings a bit.

**Review by abdonpijpelink — APPROVED** [review]
LGTM. Thanks @stefnestor !

**Comment by abdonpijpelink** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.6|[](https://github.com/elastic/elasticsearch/pull/92900)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**